### DISCUSSION TITLE
Realtime Audio Manipulations, Processing and Waveforms

### DISCUSSION THREAD

**Discussion by maxilon001** [flet-dev/flet#5114] [Extensions]
Please add the flutter equalizer and visualizer packages to Flet.

**Comment by FeodorFitsner** [comment]
Could you please provide links to the packages?

**Comment by maxilon001** [comment]
**Audio Visualizer**
[music_visualizer](https://pub.dev/packages/music_visualizer)
[flutter_audio_waveforms](https://pub.dev/packages/flutter_audio_waveforms)

**Audio Equalizer**
**Audio & Video** - [dart_vlc](https://pub.dev/packages/dart_vlc) / [media_kit](https://pub.dev/packages/media_kit)

[coast_audio](https://pub.dev/packages/coast_audio)

**Comment by maxilon001** [comment]
Please any update on this?
